frontstage/templates/register/register.enter-your-details.html

```diff
@@ -54,13 +54,19 @@ <h1 class="u-fs-xl">Enter your account details</h1>
         {{ form.enrolment_code }}
 
         <h2>Your name</h2>
-
+        {% if errors %}
+            {% set firstName = form.first_name.data %}
```

Comment:
Same with this one, Does this need to only happen if there is an error? What happens if you just get the data from an empty form?

Reply:
We need to retain the data on the UI after there is an error. So we will land up on the same view only in case of error. Data from empty form will not have this attribute set and hence will be shown blank.